report-service/src/test/kotlin/ru/morkovka/report/controller/ReleaseServiceImplTest.kt

```diff
@@ -0,0 +1,49 @@
+package ru.morkovka.report.controller
+/**
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit4.SpringRunner
+import java.util.*
+import kotlin.collections.set
+
+@RunWith(SpringRunner::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class ReleaseServiceImplTest {
+
+    @Autowired
+    private lateinit var releaseServiceImplTest: ReleaseServiceImplTest
+
+    @Test
+    fun customSort(map: MutableMap<String, MutableList<String>>) {
```

Comment:
Зачем у метода параметры на вход? Методу теста нет необходимости в точности повторять сигнатуру оригинального метода. К тому же параметр `map` имеет такое же имя как и переменная, определенная в 27 строчке, то есть значение параметра метода использоваться не сможет.
```suggestion
    fun customSortTest() {
```